app/assets/stylesheets/application.scss

```
@@ -32,9 +32,51 @@ $sidebar-width: 220px;
 // makes bootstrap-sass call Sprockets-only Sass functions (font-path/
 // image-path) that Dart Sass doesn't have -- it silently emits them as
 // literal (broken) CSS instead of erroring. Plain 'bootstrap' falls back to
-// relative paths instead; the glyphicon font files are vendored to match at
-// public/fonts/bootstrap/.
-@import 'bootstrap';
+// relative paths instead.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)